{
    "config": {
        "error": {
            "cannot_connect": "Verbindung fehlgeschlagen",
            "invalid_auth": "Ung\u00fcltige Authentifizierung",
            "unknown": "Unerwarteter Fehler"
        },
        "step": {
            "auth": {
                "data": {
                    "password": "Passwort",
                    "username": "Benutzername"
                },
                "description": "Das IoTawatt Ger\u00e4t erfordert eine Authentifizierung. Bitte gib den Benutzernamen und das Passwort ein und dr\u00fccke auf die Schaltfl\u00e4che Senden."
            },
            "user": {
                "data": {
                    "host": "Host"
                },
                "data_description": {
                    "host": "Der Hostname oder die IP-Adresse deines IoTaWatt-Ger\u00e4ts."
                }
            }
        }
    }
}